internal/app/feed/export_feeds.go

```diff
@@ -116,3 +116,20 @@ func ExportFeeds(v *viper.Viper, apiClient api.APIClient, exporter Exporter) err
 
 	return nil
 }
+
+func ExportInspectionReports(v *viper.Viper, apiClient api.APIClient, exporter *ReportExporter) error {
+	logger := util.GetLogger()
+	ctx := context.Background()
+
+	feed := GetFeeds(v)[0]
```

Comment:
Rather depending on the ordering of the array, I would suggest to figure out another way to fetch Inspection-Feed.
May be a new function to get inspection-feed ?